### PR TITLE
Add "new issue" btn on project

### PR DESCRIPTION
**Pull request by romdum**
To solve #14824.

I added a button at left of "New Project Board" to create an issue from project view. 

As suggest in issue #14824, it's possible to move "New Project Board" button at right of boards.

### CODE REVIEW

File: templates/repo/projects/view.tmpl [hunk 1]
```diff
@@ -2,12 +2,15 @@
...
+		<div class="ui two column stackable grid">
 			<div class="column">
 				{{template "repo/issue/navbar" .}}
 			</div>
 			<div class="column right aligned">
 				{{if and .CanWriteProjects (not .Repository.IsArchived) .PageIsProjects}}
+					<a class="ui green button show-modal item" href="{{$.RepoLink}}/issues/new?project={{$.Project.ID}}">{{.i18n.Tr "repo.issues.new"}}</a>
+				{{end}}
+				{{if and (not .Repository.IsArchived) .PageIsProjects}}
 					<a class="ui green button show-modal item" data-modal="#new-board-item">{{.i18n.Tr "new_project_board"}}</a>
```
Comment (silverwind) [inline templates/repo/projects/view.tmpl:12]: Why remove the `.CanWriteProjects` condition for this button?

Comment (romdum) [reply templates/repo/projects/view.tmpl:12]: I was thinking you can add an issue on a project even if you can write a project, after verification I was wrong. I fix that ;)

Comment (silverwind) [reply templates/repo/projects/view.tmpl:12]: Ok, the `if` blocks now have the same condition, so you can move the buttons below one shared block.

**Review by silverwind — APPROVED** [review]

**Review by axifive — APPROVED** [review]

**Comment by 6543** [comment]
a followup would be: after issue creation was sucessfull, redirect back to project

**Pull request merged**